### PR TITLE
Wwang talend/tdi 44172

### PR DESCRIPTION
**Pull request by wwang-talend**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.libs/libs_src/talend-job-monitoring/src/main/java/org/talend/job/audit/JobContextBuilder.java [hunk 1]
```diff
@@ -86,5 +96,59 @@ public JobContextBuilder status(String status) {
...
+        builder.with("sourceLabel", sourceLabel);
+        return this;
+    }
+    
+    /**
+     * source connector name
+     * @return
+     */
+    public JobContextBuilder sourceConnectorName(String sourceConnectorName) {
+        builder.with("sourceConnectorName", sourceConnectorName);
```
Comment (elogeart) [inline main/plugins/org.talend.designer.components.libs/libs_src/talend-job-monitoring/src/main/java/org/talend/job/audit/JobContextBuilder.java:123]: Could you rename the context key `"sourceConnectorName"` to `"sourceConnectorType"` to keep the same vocabulary as for "connectorType" please? 🙂

File: main/plugins/org.talend.designer.components.libs/libs_src/talend-job-monitoring/src/main/java/org/talend/job/audit/JobContextBuilder.java [hunk 2]
```diff
@@ -86,5 +96,59 @@ public JobContextBuilder status(String status) {
...
+        builder.with("targetLabel", targetLabel);
+        return this;
+    }
+    
+    /**
+     * target connector name
+     * @return
+     */
+    public JobContextBuilder targetConnectorName(String targetConnectorName) {
+        builder.with("targetConnectorName", targetConnectorName);
```
Comment (elogeart) [inline main/plugins/org.talend.designer.components.libs/libs_src/talend-job-monitoring/src/main/java/org/talend/job/audit/JobContextBuilder.java:150]: Could you rename the context key `"targetConnectorName"` to `"targetConnectorType"` to keep the same vocabulary as for "connectorType" please? 🙂

File: main/plugins/org.talend.designer.components.libs/libs_src/talend-job-monitoring/src/main/java/org/talend/job/audit/JobAuditLogger.java [hunk 3]
```diff
@@ -13,13 +13,16 @@
...
 
-    @AuditEvent(category = "connection", message = "Component {connectorType} {connectionType} {rows} rows in {duration} with {connectionName} line", level = LogLevel.INFO)
+    @AuditEvent(category = "connection", message = "Component {connectorType} {connectorLabel} {connectionType} {rows} rows in {duration} with {connectionName} line", level = LogLevel.INFO)
     void flowOutput(Context context);
     
-    @AuditEvent(category = "connection", message = "Component {connectorType} received {rows} rows in {duration} with {connectionName} line", level = LogLevel.INFO)
+    @AuditEvent(category = "connection", message = "Component {connectorType} {connectorLabel} received {rows} rows in {duration} with {connectionName} line", level = LogLevel.INFO)
     void flowInput(Context context);
+    
+    @AuditEvent(category = "connection", message = "connection : {connectionName}, row : {rows}, cost : {duration}", level = LogLevel.INFO)
```
Comment (elogeart) [inline main/plugins/org.talend.designer.components.libs/libs_src/talend-job-monitoring/src/main/java/org/talend/job/audit/JobAuditLogger.java:25]: I think we forget to ask you but we need some (little) changes in this interface: could you change all `category` attributes to the name of the corresponding method please ? 🙂 
I mean, current code is: 
```
@AuditEvent(category = "connection" ... ) {
    void flowExecution(...);
}
```
and we need instead: 
```
@AuditEvent(category = "flowExecution" ... ) {
    void flowExecution(...);
}
```
and so on for each `@AuditEvent`.

Comment (wwang-talend) [reply main/plugins/org.talend.designer.components.libs/libs_src/talend-job-monitoring/src/main/java/org/talend/job/audit/JobAuditLogger.java]: fixed

File: main/plugins/org.talend.designer.components.libs/libs_src/talend-job-monitoring/src/main/java/org/talend/job/audit/JobContextBuilder.java (hunk 2 above)
Comment (wwang-talend) [reply main/plugins/org.talend.designer.components.libs/libs_src/talend-job-monitoring/src/main/java/org/talend/job/audit/JobContextBuilder.java]: fixed

**Pull request merged**